vscode-dotnet-runtime-library/install scripts/interprocess-communicator.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+EXECFOLDER=$1 # First argument is the working folder as this is launched with cwd of /root
+OKSIGNALFILE="$EXECFOLDER/ok.txt"
+COMMANDTORUNFILE="$EXECFOLDER/command.txt"
+OUTPUTFILE="/home/test_output_.txt"
+end=$((SECONDS+3600))
+while true
+do
+        stop=false
+        while [ $SECONDS -lt $end ];
+        do
+            if test -f "$COMMANDTORUNFILE"; then
+                # echo "COMMAND FILE FOUND" >> "$OUTPUTFILE" # Leave this here as an example of debugging
+                COMMAND="$(cat "$COMMANDTORUNFILE" | awk '{$1=$1;print}')"
+                for validCmd in "${@:2}"
+                do
+                    if [ "$COMMAND" == "$validCmd" ]; then
+                        IFS=' ' read -ra COMMANDARGS <<< "$COMMAND"
+                    fi
+                done
+                if [ -z "$COMMANDARGS" ]; then
+                    rm "$COMMANDTORUNFILE"
```

Comment:
I see this cleanup-style stuff in multiple places, consider using a [bash 'exit trap' ](http://redsymbol.net/articles/bash-exit-traps/) as a way to consolidate all of the cleanup. This will also help if someone terminates the process outside of your control. This could cover removing the ok file and the command to run file at minimum.

Reply:
Oh, that's pretty handy. Thank you!